### PR TITLE
[MIG][13.0] stock_barcodes_gs1: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
TT22177

### CODE REVIEW

**Comment by jimhoefnagels** [comment]
The dependency is merged, so rebasing and running pre-commit should fix the errors.

**Comment by sergio-teruel** [comment]
Sorry @jimhoefnagels stock_barcodes module is still open... 
https://github.com/OCA/stock-logistics-barcode/pull/252

**Comment by AaronHForgeFlow** [comment]
https://github.com/OCA/stock-logistics-barcode/pull/252 is merged :tada:  could you rebase?

**Comment by sergio-teruel** [comment]
@AaronHForgeFlow rebase done!!

**Comment by sergio-teruel** [comment]
Ready!!

File: stock_barcodes_gs1/tests/test_stock_barcodes_gs1.py [hunk 1]
```diff
@@ -19,10 +22,10 @@ def setUp(self):
             "01993167101234533101002620130" "5041710ABC123214145354"
         )
 
-        self.product_wo_tracking_gs1 = self.product_wo_tracking.copy(
+        self.product_wo_tracking_gs1 = self.product_wo_tracking.with_context({}).copy(
```
Comment (rousseldenis) [inline stock_barcodes_gs1/tests/test_stock_barcodes_gs1.py:25]: Why this ?

Comment (sergio-teruel) [reply stock_barcodes_gs1/tests/test_stock_barcodes_gs1.py:25]: In the test I am copying a product.product, if I do not reset context the product.template does not create the variant...
Is better solution (only for test that @Tardo applied in this case)

**Comment by sergio-teruel** [comment]
@carlosdauden @chienandalu CAn you review... please??

**Review by chienandalu — APPROVED** [review]
Tested on runbot and code review :+1:

**Review by rafaelbn — APPROVED** [review]
:+1:  Thanks

**Comment by pedrobaeza** [comment]
Is this still WIP?

**Comment by sergio-teruel** [comment]
@pedrobaeza ready!!

**Pull request merged**